src/lr/lr.php

```diff
@@ -4,23 +4,42 @@
 	header("Location: ../login.php");
 }
 require_once("../php/connect.php");
-$usernow = mysql_query("SELECT * FROM `user` WHERE `id` = '{$_SESSION['user']}'");
+$stmt = $conn->prepare("SELECT * FROM `user` WHERE `id` = ?");
+$stmt->bind_param("i", $_SESSION['user']);
+$stmt->execute();
+$usernow = $stmt->get_results();

```

Comment:
`get_results` does not work, please consider replacing with `get_result` instead

Reply:
Solved in the latest commit.